webpack.config.js

```diff
@@ -20,7 +20,13 @@ var config = {
     extensions: [".webpack.js", ".web.js", ".ts", ".tsx", ".js"],
     modules: [
       "node_modules",
-      "src"]
+      "src",
+    ]
+  },
+  node: {
+    fs: "empty",
```

Comment:
The thing is that we have to generate two js distribution because WS only works on nodeJs. So i'am trying to test signalr js or other library to avoid that. I choose WS because it is quite similar to window.websocket which won't modify the code that much.